generators/service-watson-conversation/templates/python/README.md

```diff
@@ -0,0 +1,54 @@
+# Watson Conversation
+
+The IBM Watson™ Conversation service combines machine learning, natural language understanding, and integrated dialog tools to create conversation flows between your apps and your users.
+
+##  Credentials
+
+###  LocalDevConfig
+
+This is where your local configuration is stored for MongoDB.
```

Comment:
I assume you mean to say Watson Conversation here instead of MongoDB?

Reply:
Yes, good catch.